v2/hack/verify-examples.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# hack/verify-examples.sh
```

Comment:
Just thinking ahead for a moment..

We should tool our testing infra to run v2/hack/verify-examples.sh? We could run it from the root `hack/verify-examples.sh` but that running it separately will probably result in clearer error reporting.

Reply:
Good point!